templates/hotgridPopupToolbar.jsx

```diff
@@ -0,0 +1,80 @@
+import Adapt from 'core/js/adapt';
+import React from 'react';
+import { classes } from 'core/js/reactHelpers';
+
+export default function HotgridPopupToolbar(props) {
+  const a11yConfig = Adapt.config.get('_accessibility');
+  const ariaLabels = Adapt.course.get('_globals')._accessibility._ariaLabels;
+
+  const {
+    onCloseClick,
+    onControlClick,
+    shouldEnableBack,
+    shouldEnableNext,
+    itemCount,
+    _hidePagination
+  } = props;
+
+  return (
+    <div className={classes([
+      'hotgrid-popup__toolbar',
+      _hidePagination && 'hide-pagination'
+    ])}>
+
+      {!_hidePagination &&
+      <div className={classes([
+        'hotgrid-popup__nav',
+        !shouldEnableBack && 'first',
+        !shouldEnableNext && 'last'
+      ])}
+      >
+
+        <button
+          className={classes([
+            'btn-icon hotgrid-popup__controls back',
+            !shouldEnableBack && 'is-disabled'
+          ])}
+          aria-label={ariaLabels.previous}
+          aria-disabled={!shouldEnableBack}
+          onClick={onControlClick}
+          data-direction='back'
+        >
+          <span className="icon" />
+        </button>
+
+        <div
+          className="hotgrid-popup__count"
+          dangerouslySetInnerHTML={{ __html: itemCount }}
+        />
+
+        <button
+          className={classes([
+            'btn-icon hotgrid-popup__controls next',
+            !shouldEnableNext && 'is-disabled'
+          ])}
+          aria-label={ariaLabels.next}
```

Comment:
Aira-label needs to be much more detailed here so should say - 'Forward to item 2 (item 2 of 3)'.

I think the hotgraphic follows this convention but says - 'Forward to Hot Graphic item 2 (item 2 of 3)' - Im not sure 'hot-graphic' is necessary there, may need to raise an issue on thee hot-graphic repo.

In general though the pop -up should be wcag compliant and follow the carousel behaviour here:

https://www.w3.org/WAI/tutorials/carousels/

This has already been implemented got hot-graphic here:

https://github.com/adaptlearning/adapt-contrib-hotgraphic/pull/244

So you can use that as a reference to check. Happy to have this one in a separate PR if need be

Reply:
Noting that this fix will close #89.

Reply:
This is ready for review.  Requires the schema updates from #94 .

I noticed that Hot Graphic's controls don't seem to take into account `_canCycleThroughPagination` being enabled.  So, I've adapted and modified the logic for Hot Grid.  Probably should be fixed in Hot Graphic, though. So, when `_canCycleThroughPagination` is false and the back or next button is disabled, the aria-label will simply read "back" or "next" respectively.  Otherwise, it will correctly say what the next or previous item number is.

> I think the hotgraphic follows this convention but says - 'Forward to Hot Graphic item 2 (item 2 of 3)' - Im not sure 'hot-graphic' is necessary there, may need to raise an issue on thee hot-graphic repo.

@StuartNicholls This uses the item's title rather than automatically including the "Hot Graphic" plugin name.  Are you saying we don't need the title at all and should just use "forward to item 2"?

Reply:
My bad! Title needs to stay :)

Reply:
Looks like I broke the popup focus.  It should always focus on the notify title not the back button.  Will investigate tomorrow.

Reply:
^^ should be fixed.